project.clj

```diff
@@ -20,15 +21,19 @@
             [lein-cljsbuild "1.1.7" :exclusions [org.clojure/clojure
                                                  org.clojure/clojurescript]]
             [lein-figwheel "0.5.14" :exclusions [org.clojure/clojure
-                                                 org.clojure/clojurescript]]]
+                                                 org.clojure/clojurescript]]
+            [com.github.clj-kondo/lein-clj-kondo "0.2.4" :exclusions [org.clojure/clojure
+                                                                      org.clojure/clojurescript]]]
+  :aliases {"clj-kondo-deps" ["clj-kondo" "--copy-configs" "--dependencies" "--parallel" "--lint" "$classpath"]
+            "clj-kondo-lint" ["do" ["clj-kondo-deps"] ["clj-kondo" "--lint" "src/main" "--fail-level" "error"]]}
   :codox {:namespaces [clara.rules clara.rules.dsl clara.rules.accumulators
                        clara.rules.listener clara.rules.durability
                        clara.tools.inspect clara.tools.tracing
                        clara.tools.fact-graph]
           :metadata {:doc/format :markdown}}
   :javadoc-opts {:package-names "clara.rules"}
   :source-paths ["src/main/clojure"]
-  :resource-paths []
+  :resource-paths ["clj-kondo"]
```

Comment:
This seems like it should only be in the dev profile and not the top level where it’ll become part of the deployed jar.

Reply:
This was done on purpose, in order to export the hooks as built-in with the jar they need to be bundled, and I didn't want to bundle `"resources"` because the JS is being built there and those are excluded on purpose.

Exporting a configuration in a dependency:
https://github.com/clj-kondo/clj-kondo/blob/master/doc/config.md#exporting